gradle.properties

```diff
@@ -4,3 +4,4 @@ kotlin.mpp.enableGranularSourceSetsMetadata=true
 kotlin.native.enableDependencyPropagation=false
 systemProp.org.gradle.internal.publish.checksums.insecure=true
 android.useAndroidX=true
+org.gradle.jvmargs=-Xmx2048m
```

Comment:
You should not commit memory based settings. These should be stored in your `~/.gradle/gradle.properties` file, outside of the repository. This allows each machine to specify their needs for building.